### ISSUE TITLE
compass-rails (= 2.0.4) ruby depends on sprockets (< 2.13) ruby

### ISSUE THREAD

**Issue by jonricaurte** [Compass/compass-rails#232]
Trying to use the latest version of sprockets which is 3.0.0.rc.2 and get the following error:

compass-rails (= 2.0.4) ruby depends on
      sprockets (< 2.13) ruby

Thanks.

**Comment by tagliala** [comment]
PR #230 addresses this issue.

I'm currently using
```rb
gem 'compass-rails', github: 'y-yagi/compass-rails', branch: 'sass-rails-5_0_x'
```

**Comment by stefanahman** [comment]
@tagliala It is still depending on sprockets 2.x:
```ruby
  gem.add_dependency 'compass', '~> 1.0.0'
  gem.add_dependency 'sprockets', '< 2.13'
  gem.add_dependency 'sass-rails', '< 5.1'
```

**Comment by tagliala** [comment]
@stefanahman thanks, I was thinking to something else. Maybe to sass, don't remember

Edited my previous comment

**Comment by JacobEvelyn** [comment]
@tagliala did you find a solution? I too want to use the latest version of `sprockets`.

**Comment by PikachuEXE** [comment]
I guess need compass team open a branch or yourself to fork the repo
Official support properly won't come soon

PS what's new in sprockets 3?

**Comment by stefanahman** [comment]
@PikachuEXE https://github.com/sstephenson/sprockets/issues/643

**Comment by knightq** [comment]
+1

**Comment by woohoou** [comment]
+1

**Comment by edword** [comment]
+1

**Comment by djforth** [comment]
+1

**Comment by nickbouton** [comment]
+1

**Comment by mkluczny** [comment]
+1

**Comment by PikachuEXE** [comment]
Please reply only if you have something to add.
Comments with only +1 won't make it faster to be finished, but make other people harder to read the whole issue.

**Comment by will-r** [comment]
Recent versions of capistrano assume that a Sprockets 3.x manifest will be present, so compass-rails is getting quite hard to use. We found that the easy solution was to strip compass out of our rails projects. It's really not that useful any more.

**Comment by reconstructions** [comment]
@PikachuEXE I would like to add that none of these comments would even have been made if the README was simply updated to indicate the exact versions of the dependencies which worked with this gem.  It has been more than a year since sprockets had to be locked to use this gem and the current README guarantees that every single installation of this gem will go wrong.

**Comment by francois2metz** [comment]
Fixed now. Should be closed since #246 has been merged and the 3.0 released.
